portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,23 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1 id = 'port_section'>Cindy's Portfolio</h1>
```

Comment:
didn't notice this during the first review, but please follow the google html/css style guide which includes [using double quotes for attribute values](https://google.github.io/styleguide/htmlcssguide.html#HTML_Quotation_Marks). Also I can'
t find it in the style guide explicitly, but using a space between the assignment operator, `=`, and the attribute and attribute value seems strange.